Replace discussion of BUNDLE (no longer useful with RoQ)

SpencerDawkins opened the pull request:
closes #3

After some discussion, @mengelbart and I were unable to develop a coherent theory of why BUNDLE would be used with RoQ, since the primary benefit of BUNDLE (multiple media flows on a single transport connection) is already provided by RoQ (multiple media flows on a single QUIC connection).

I updated the SDP example (while correcting some mistakes) to show multiple media flows on a single RoQ connection, as @JonathanLennox had suggested in email (I was too lazy to do that, when I was putting the initial SDP example together).  That seemed closely related to the work I was already doing in this PR. 

Thank you, @srinivasgu, for pointing out the mistakes during the June 10 virtual interim meeting.

SpencerDawkins committed:
Update draft-dawkins-avtcore-sdp-roq.md

closes #3

After some discussion, @mengelbart and I were unable to develop a coherent theory of why BUNDLE would be used with RoQ, since the primary benefit of BUNDLE (multiple media flows on a single transport connection) is already provided by RoQ (multiple media flows on a single QUIC connection).

I updated the SDP example (while correcting some mistakes) to show multiple media flows on a single RoQ connection, as @JonathanLennox had suggested in email (I was too lazy to do that, when I was putting the initial SDP example together).  That seemed closely related to the work I was already doing in this PR.

alvestrand commented on draft-dawkins-avtcore-sdp-roq.md:
```diff
@@ -250,15 +250,15 @@ Because QUIC itself uses the TLS handshake as described in {{!RFC9001}}, the par
+For this reason, it is assumed that RTP applications being ported to use RoQ with SDP will no longer use BUNDLE, but will be updated to use RoQ Flow IDs for multiplexing, with the appropriate changes being made in SDP.
 
-If an application uses multiple 5-tuples in order to make use of the Multipath Extension for QUIC as described in {{?I-D.draft-ietf-quic-multipath}}, this would allow multiple QUIC paths to be active simultaneously, and this assumption will need revisiting when {{?I-D.draft-ietf-quic-multipath}} is approved.
+RoQ applications wishing to use multiple transport connections for media simply use multiple QUIC connections, each with its own connection information ("c="), with each QUIC connection carrying one or more media flows, each identified with its own RoQ Flow ID, described at the media description level in SDP.
```
Multiple QUIC connections makes no deployment sense - it has all the issues that caused BUNDLE to be a requirement for WebRTC deployment.

SpencerDawkins committed:
Significantly discourage the use of QUIC connection migration (as opposed to RTP-level path migration).

This is based on review comments based on a previous version of the PR from @alvestrand (for which, thank you, Harald).

SpencerDawkins committed:
Provide guidance about RoQ/non-RoQ translation

Again, thank you, @alvestrand, for the pointer!

alvestrand reviewed (commented):
Note: ROQ still refers to BUNDLE.
A deeper issue is what RTP sessions are described here - is each media section a separate RTP session, or do all the media sections form one RTP session?
(This is a leftover from when RTP was for multicast....)

This matters because of feedback messages - for instance, RFC8888 feedback carries info for multiple SSRCs - on which ROQ ID it should be sent should be clear, somehow.

SpencerDawkins commented:
This PR replaced by #35, so closed now.
